internal/backends/sqlite/database.go

```diff
@@ -105,9 +105,9 @@ func (db *database) RenameCollection(ctx context.Context, params *backends.Renam
 
 // Stats implements backends.Database interface.
 //
-// If the database does not exist, it returns *backends.StatsResult filled with zeros for all the fields.
-func (db *database) Stats(ctx context.Context, params *backends.StatsParams) (*backends.StatsResult, error) {
-	stats := new(backends.StatsResult)
+// If the database does not exist, it returns *backends.DatabaseStatsResult filled with zeros for all the fields.
```

Comment:
That comment should be in the contract only. All comments in implementations should be "XXX Stats implements YYY interface." to avoid keeping them in sync across multiple files